docs/video/obs/quicksync.md

```diff
@@ -3,4 +3,31 @@ description: OBS configuration for Intel GPUs
 icon: simple/intel
 ---
 
-# :simple-intel: QuickSync OBS Configuration
\ No newline at end of file
+# :simple-intel: QuickSync OBS Configuration
+
+## :material-cog: Configuration
+
+=== "Medium Quality & Low FPS"
```

Comment:
the preset names should be kept consistent really, so we'd have:
- Maximum Performance 
- Best Filesize
- Best Quality
- Streaming

for each encoder imo